src/utilities/FormRUtilities.tsx

```diff
@@ -117,6 +119,17 @@ export class FormRUtilities {
     return value ? value : message;
   }
 
+  public static dispaySubmissionDate(date: DateType) {
```

Comment:
Also, I think you have a typo in util method name:  `dispaySubmissionDate` (missing an 'l'?)